napari/utils/theme.py

```diff
@@ -220,12 +220,7 @@ def get_theme(name, as_dict=None):
         )
         as_dict = True
     if as_dict:
-        _theme = _theme.dict()
-        _theme = {
-            k: v if not isinstance(v, Color) else v.as_rgb()
-            for (k, v) in _theme.items()
-        }
-        return _theme
+        return dict(_theme)
```

Comment:
Assuming this works, do you think `as_dict` should be deprecated ?

Reply:
It may be deprecated. I wrote it to keep backward compatibility (one code for multiple napari version). 

The core problem is to allow simple apply  style to qt widgets that does not have viewer in parent line.

I have plan to add new function that will return current style sheet. To reduce multiple calls required now to one.

Reply:
Yes can we please deprecate and remove. I recall when we did this and wanted to add the deprecation, but it doesn't look like we did

Reply:
#4780 has my idea of implementing one function for the current theme.